pkg/operator/cassandra/controller/util/resource.go

```diff
@@ -293,7 +290,13 @@ func volumeMountsForRack(r cassandrav1alpha1.RackSpec, c *cassandrav1alpha1.Clus
 			ReadOnly:  true,
 		},
 	}
-
+	if *r.JMXExporterConfigMapName != "" {
```

Comment:
This should be `if r.JMXExporterConfigMapName != nil && *r.JMXExporterConfigMapName != ""`

Reply:
Thanks, fixed in both places!